packages/pn-personafisica-webapp/src/component/OutlinedButton.tsx

```diff
@@ -0,0 +1,16 @@
+import { Button, styled } from '@mui/material';
```

Comment:
Questo componente potrebbe essere un candidato per entrare in pn-commons

Reply:
Posso spostarli direttamente in pn-commons o vogliamo prima discuterne nel meet di oggi pomeriggio?

Reply:
per me puoi spostare senza problemi